indexing-service/src/main/java/io/druid/indexing/overlord/WorkerTaskRunner.java

```diff
@@ -18,37 +18,7 @@
  */
 package io.druid.indexing.overlord;
 
-import com.google.common.base.Predicate;
-import io.druid.indexing.common.task.Task;
-import io.druid.indexing.overlord.config.WorkerTaskRunnerConfig;
-import io.druid.indexing.worker.Worker;
 
-import java.util.Collection;
-
-public interface WorkerTaskRunner extends TaskRunner
+public interface WorkerTaskRunner extends TaskRunner, TasksAndWorkers
```

Comment:
@clintropolis `WorkerTaskRunner` also extends `TaskRunner`, which contains a lot of methods which I didn't want to implement in `TasksAndWorkersFilteredByIp`.